inspirehep/modules/records/utils.py

```diff
@@ -96,7 +96,7 @@ def get_pid_from_record_uri(record_uri):
 def get_author_display_name(name):
     """Returns the display name in format Firstnames Lastnames"""
     parsed_name = ParsedName.loads(name)
-    return " ".join(parsed_name.first_list + parsed_name.middle_list + parsed_name.last_list)
+    return " ".join(parsed_name.first_list + parsed_name.last_list)
```

Comment:
this might break https://github.com/inspirehep/inspire-next/blob/a9e47eab306a26df7ffa429765da8d95f95b9d23/inspirehep/modules/records/utils.py#L409

Reply:
:+1: this should have broken a test though :cry: @vbalbp can you add a test for it and fix it?

Reply:
``ParsedName.first_list`` has been changed on inspire_utils to return was previously was ``ParsedName.first_list + ParsedName.middle_list``, since we don't make a distinction between first and middle names in INSPIRE.